src/App.js

```diff
@@ -0,0 +1,127 @@
+import axios from "axios";
+import Header from "./components/Header/Header";
+import Banner from "./components/Banner/Banner";
+import './style/main.css';
+
+import React from 'react';
+
+class App extends React.Component{
```

Comment:
I think the main component should be written as a functional component according to  one of the previous tasks